deploy/operator/monitoring-stack-operator-service-monitor.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: monitoring-stack-operator
+  labels:
```

Comment:
I wonder if we can make use of `commonLabels` in `kustomize.yaml` ?

Reply:
ya we can, I thought to have it in a separate PR since I need to update other files also  unrelated to this  change

Reply:
The problem with commonLabels is that it sets selectors as well. So we have to make sure we don't have any values that change (e.g. the version) in those labels.

Reply:
ya we can keep component, name and part-of?

Reply:
I think we can only have `part-of`, no? Otherwise every component will have the same name and same component.